docs/how-to/logs.rst

```diff
@@ -0,0 +1,62 @@
+==============
+Accessing logs
+==============
+
+The :ref:`MicroOVN services` provide logs as part of their normal operation.
+
+By default they are provided through the systemd journal.
+
+Below you can view an example for how to access the logs of the
+``microovn.chassis`` service:
+
+.. code-block:: none
+
+   journalctl -u microovn.chassis
+
+Below you can view an example of how to view a live log display for the same
+service:
+
+.. code-block:: none
+
+   journalctl -f -u microovn.chassis
+
+Log files
+---------
+
+Inside the ``/var/snap/microovn/common/logs`` directory you will find files for
+each individual service, however these will either be empty or not contain
+updated information, this is intentional.
+
+On fresh install the files are created, as a precaution, in the event a need
+arises for enabling `debug logging`_.  On refresh from older versions of
+MicroOVN existing files will be retained, but not updated.
+
+Debug logging
+-------------
+
+The Open vSwitch (OVS) and Open Virtual Network (OVN) daemons have a rich set
+of debug features, one of which being available by augmenting log levels for
+individual modules at run time.
+
+A list of modules can be acquired through the ``microovn.ovs-appctl`` and
+``microovn.ovn-appctl`` command line utilities, for example:
+
+.. code-block:: none
```

Comment:
is the output to these two list commands extensive? generally it is useful to include output to commands if the output is reasonable

Reply:
There's a lot of modules that can be individually configured. It outputs ~120 lines

Reply:
I agree with @mkalcok the output is quite long/verbose. Would a short/edited version of it be valuable just to get a feel of it, alternatively we could add a reference page with this information and link to that?

Reply:
(I guess an alternative could also be to omit the code block for those two commands)

Reply:
Often we do stuff like this:

```
first line output
second line output
...
penultimate line output
last line output
```

Reply:
@pmatulis The examples of `vlog/get` commands got taken out, the doc just generically points user towards `microovn.ovs-appctl `and `microovn.ovn-appctl` commands, which I think is fine. We don't need to re-document usage of the general OVN/OVS commands.
Examples of how to `vlog/set` values are still in there but they don't produce any output.

Is the current form OK with you?